packages/gatsby/migrations.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
We need a reference to this file like this:

https://github.com/nrwl/nx/blob/5410b63f8447aa2677673890d2db474492402ec2/packages/linter/package.json#L25

Reply:
[Looks like it's already there](https://github.com/nrwl/nx/blob/master/packages/gatsby/package.json#L29)

Reply:
Oh :eyes: